specs/lib/removeUnusedSkinsSpec.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+var removeUnusedSkins = require('../../').removeUnusedSkins;
+var OptimizationStatistics = require('../../').OptimizationStatistics;
+
+describe('removeUnusedSkins', function() {
+    it('removes a skin', function() {
+        var gltf = {
+            "nodes": {
+                "Cylinder": {
+                    "children": [],
+                    "matrix": [
```

Comment:
Same comment as in the other PR to remove optional properties from each `node` in these tests.